modules/build/src/main/scala/scala/build/preprocessing/mdsandbox/SnippetPackager.scala

```diff
@@ -0,0 +1,147 @@
+package scala.build.preprocessing.mdsandbox
+
+import MarkdownSnippet.Fence
+
+class SnippetPackager(val fileName: String, val snippets: Seq[Fence]) {
```

Comment:
naming it as a `packager` seems unfortunate, since it's got nothing to do with how we package things with the `package` command.
It's really just a tool for extracting the snippets from markdown, so maybe something like:
```suggestion
class MarkdownSnippetUtil(val fileName: String, val snippets: Seq[Fence]) {
```
`-Util` files in `scala-cli` are usually objects which don't require instantiating, so maybe we need an even better name

Reply:
actually, this could be a `-Util` class, since I don't think any of the methods truly require a state. the `fileName` and `snippets` could instead be passed around to static methods on an object.

in other words:
```suggestion
object MarkdownSnippetUtil {
```